time/sorted-immutable-list.d.ts

```diff
@@ -0,0 +1,2 @@
+declare module 'sorted-immutable-list';
```

Comment:
I would put them all into one `custom-typings.d.ts`

Reply:
I was thinking about that, but wasn't sure if it would work, but I agree and have done it now. 😄